packages/sources/svelte/src/components/navigation/VtmnBreadcrumbItem/VtmnBreadcrumbItem.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  import VtmnIcon from '../../../guidelines/iconography/VtmnIcon/VtmnIcon.svelte';
+
+  /**
+   * The icon to display on the left side of the breadcrumb.
+   * @type {string}
+   */
+  export let icon = undefined;
+
+  /**
+   * The href that makes the tag interactive.
+   * @type {string}
+   */
+  export let href = undefined;
+</script>
+
+<li {...$$restProps}>
+  {#if icon}
+    <VtmnIcon value={icon} aria-hidden="true" />
```

Comment:
Is it possible to set the size of the icon at 16px, even if a css class will override the property please ?